snuba/clickhouse/formatter.py

```diff
@@ -82,3 +84,11 @@ def visitCurriedFunctionCall(self, exp: CurriedFunctionCall) -> str:
         int_func = exp.internal_function.accept(self)
         ret = f"{int_func}{self.__visit_params(exp.parameters)}"
         return self.__alias(ret, exp.alias)
+
+    def visitVariable(self, exp: Variable) -> str:
+        return escape_identifier(exp.name)
+
+    def visitLambda(self, exp: Lambda) -> str:
+        variables = [escape_identifier(v) for v in exp.variables]
```

Comment:
~Doesn't this circumvent `visitVariable` by not calling `accept`?~

I initially misinterpreted this, the implementation makes sense to me now. This is an extreme nit, but I think it might be clearer to refer to these as "parameters" instead of "variables".

Reply:
Are you sure about this ? I used the term `variable` to follow the convention generally used when describing lambdas in languages that support them (which I believe comes from the formal definition in lambda calculus https://en.wikipedia.org/wiki/Lambda_calculus#Free_and_bound_variables). 
If this sounds confusing, what about `argument` so there is no naming conflict with the parameters of the FunctionCall class ?

Reply:
> Are you sure about this ?

Not entirely. I'm not going to block on it by any means.

My understanding has generally been that `parameters` are the unbound variables of the function definition (incl. lambda functions), and `arguments` are the values of the bound variables. Python does refer to them as parameters in the language definition: https://docs.python.org/3/reference/expressions.html#lambda

> If this sounds confusing, what about argument so there is no naming conflict with the parameters of the FunctionCall class ?

I think this is more confusing, to be honest. In retrospect, if my understanding is correct, `FunctionCall.parameters` should have been `arguments`.